engine/src/main/java/org/terasology/entitySystem/entity/internal/PojoEntityPool.java

```diff
@@ -81,23 +78,10 @@ public EntityRef create(Iterable<Component> components) {
 
     @Override
     public EntityRef create(Iterable<Component> components, boolean sendLifecycleEvents) {
-        components = (components == null) ? Collections.EMPTY_LIST : components;
-        EntityRef entity = createEntity(components);
-
-        if (sendLifecycleEvents) {
-            EventSystem eventSystem = entityManager.getEventSystem();
-            if (eventSystem != null) {
-                eventSystem.send(entity, OnAddedComponent.newInstance());
-                eventSystem.send(entity, OnActivatedComponent.newInstance());
-            }
-        }
-
-        //Retrieve the components again in case they were modified by the previous events
-        for (Component component : entityManager.iterateComponents(entity.getId())) {
-            entityManager.notifyComponentAdded(entity, component.getClass());
-        }
-
-        return entity;
+        EntityBuilder builder = newBuilder();
```

Comment:
Can't these calls on the builder be chained (maybe this is not so common in Java, I might got confused with Scala here)?